web3/shh.py

```diff
@@ -1,79 +1,179 @@
+
+
+def info():
+    return Method(
+        "shh_info",
+        mungers=None,
+    )
+
+
+def setMaxMessageSize():
+    return Method(
+        "shh_setMaxMessageSize",
+        mungers=[default_root_munger],
+    )
+
+
+def setMinPoW():
+    return Method(
+        "shh_setMinPoW",
+        mungers=[default_root_munger],
+    )
+
+
+def markTrustedPeer():
+    return Method(
+        "shh_markTrustedPeer",
+        mungers=[default_root_munger],
+    )
+
+
+def newKeyPair():
+    return Method(
+        "shh_newKeyPair",
+        mungers=None,
+    )
+
+
+def addPrivateKey():
+    return Method(
+        "shh_addPrivateKey",
+        mungers=[default_root_munger],
+    )
+
+
+def deleteKeyPair():
+    return Method(
+        "shh_deleteKeyPair",
+        mungers=[default_root_munger],
+    )
+
+
+def deleteKey():
+    return Method(
+        "shh_deleteKey",
+        mungers=[default_root_munger],
+    )
+
+
+def hasKeyPair():
+    return Method(
+        "shh_hasKeyPair",
+        mungers=[default_root_munger],
+    )
+
+
+def getPublicKey():
+    return Method(
+        "shh_getPublicKey",
+        mungers=[default_root_munger],
+    )
+
+
+def getPrivateKey():
+    return Method(
+        "shh_getPrivateKey",
+        mungers=[default_root_munger],
+    )
+
+
+def newSymKey():
+    return Method(
+        "shh_newSymKey",
+        mungers=None,
+    )
+
+
+def addSymKey():
+    return Method(
+        "shh_addSymKey",
+        mungers=[default_root_munger],
+    )
+
+
+def generateSymKeyFromPassword():
+    return Method(
+        "shh_generateSymKeyFromPassword",
+        mungers=[default_root_munger],
+    )
 
-    @property
-    def info(self):
-        return self.web3.manager.request_blocking("shh_info", [])
 
-    def setMaxMessageSize(self, size):
-        return self.web3.manager.request_blocking("shh_setMaxMessageSize", [size])
+def hasSymKey():
+    return Method(
+        "shh_hasSymKey",
+        mungers=[default_root_munger],
+    )
 
-    def setMinPoW(self, min_pow):
-        return self.web3.manager.request_blocking("shh_setMinPoW", [min_pow])
 
-    def markTrustedPeer(self, enode):
-        return self.web3.manager.request_blocking("shh_markTrustedPeer", [enode])
+def getSymKey():
+    return Method(
+        "shh_getSymKey",
+        mungers=[default_root_munger],
+    )
 
-    def newKeyPair(self):
-        return self.web3.manager.request_blocking("shh_newKeyPair", [])
 
-    def addPrivateKey(self, key):
-        return self.web3.manager.request_blocking("shh_addPrivateKey", [key])
+def deleteSymKey():
+    return Method(
+        "shh_deleteSymKey",
+        mungers=[default_root_munger],
+    )
 
-    def deleteKeyPair(self, id):
-        return self.web3.manager.request_blocking("shh_deleteKeyPair", [id])
 
-    def hasKeyPair(self, id):
-        return self.web3.manager.request_blocking("shh_hasKeyPair", [id])
+def post_munger(module, message):
+    if message and ("payload" in message):
+        return (message,)
```

Comment:
@dylanjw I'm not sure if this is a misunderstanding on my end or a bug with the munger, but this was the only way I could successfully use this munger. From what I understand this should return something like `return module, [message]` - but that wouldn't work.

Reply:
mungers only need to return the arguments, and not the module object.  This was changed, and it looks like I didnt update the docstring in the Method class.

So what you have here is correct.